stored/src/components/navbar/AuthenticatedNavbarButtons.tsx

```diff
@@ -0,0 +1,39 @@
+import { SessionContextValue, signOut } from "next-auth/react";
+import Link from "next/link";
+import { FaUser } from "react-icons/fa";
+
+export type AuthenticatedNavbarButtonsProps = {
+  session: SessionContextValue<true>;
+  mobileView?: boolean;
+};
+
+export default function AuthenticatedNavbarButtons({
+  session,
+  mobileView,
+}: AuthenticatedNavbarButtonsProps) {
+  const linkClassName = `text-white ${
```

Comment:
Since Tailwind is a bit tricky it's better to avoid template literals and conditional class names. While this linkClassName might work ok a safer approach would be something like: className={cn(
  "text-white hover:bg-white hover:text-black rounded-lg p-2 dark:text-indigo-100 dark:hover:bg-indigo-200 dark:hover:text-neutral-800",
  mobileView && 'block',
)}

Reply:
I have to update my own code as well with the same practice 😃